### PR TITLE
ready for PR

### LINKED ISSUE #46: Make any PVS conversion work Optional
It would be great to be able to continue with the BIS-F Routines even IF the PVS Target is installed, but we DON'T want BIS-F to do any conversion....A setting to say do nothing would be great

### PR DESCRIPTION
**Pull request by andre-sicking**
Thanks for submitting a pull request! Please provide enough information so that others can review your pull request:

Added ADMX/ADML configuration option that allows for skipping P2V process completely and just preparing a base disk even if Provisioning Services Target Device software is installed.

Please feel free to change anything (wordings etc.)

I successfully tested this change with all 8 combinations of the following settings:
- PVS RW Target Device disk attached, yes/no
- Skip image creation checked in group policy, yes/no
- UNC-Path for offline image creation set and active, yes/no

Some typos were also fixed.

I'm planning some more changes in the future to further enhance PVS Target Device options (i.e. free space check) but I'm using this as a first pull request as I'm still new to GitHub (but not to PowerShell and BIS-F).

The two files 98_PrepBISF_BUILD_BaseImage.ps1 and 99_PrepBISF_POST_BaseImage.ps1 are formatted UTF-16 LE in master branch which I did not change. Sadly this leads to GitHub interpreting them as binary files. Maybe these files could be converted to UTF-8 throughout the project? The various files seem to differ a lot speaking of the file format used.

If you have any questions you can contact me here or on twitter (@andre_sicking). German or English language would be fine. If you would have to reject my pull request due to any reason please leave me a note of what was wrong so I can fix it (and for future requests).

Does the code pass AppVeyor?
* [ ] Yes

Not tested in AppVeyor. Used own environment due to PVS Server requirement. OS used for testing: Windows Server 2016 German fully patched.

**Closing issues**

closes #46

### CODE REVIEW

**Comment by EUCweb** [comment]
Thx Andre for sending the pull request. For a short overview great idea but I don’t see the difference between the existing UNC Path integration, checkout https://eucweb.com/docs/bis-f/configuration/citrix <https://eucweb.com/docs/bis-f/configuration/citrix> Configure Citrix PVS Target Device.  Can you give me a short explanation of the difference ?


For your understanding Pull requests are accepted to this branch only https://github.com/EUCweb/BIS-F/tree/BB_develop <https://github.com/EUCweb/BIS-F/tree/BB_develop>, this is the upcoming new release planned for end of this year

**Comment by andre-sicking** [comment]
>Can you give me a short explanation of the difference ?
If PVS Target Device software is installed and you want BIS-F to configure PVS (i.e. set writecache drive to D: etc.) you have 2 options: Let BIS-F copy the image to an attached PVS vDisk in Private Mode afterwards or do an offline imaging to an UNC path. But either one of them has to be done to get proper PVS integration (as far as I understand). If you want to skip the whole imaging process (for testing purpose of image building, i.e. quick MCS usage of the image with PVS target device software installed but staying inactive) you only have the time consuming option of doing the copy to an UNC path.

I think it's more convenient to be able to skip this process if one wants to.

Maybe I should clarify (in the ADML files) that this in most cases might be a choice for mainly testing purposes? There were other users (see #46) that already were requesting this and I think it's a good idea if you really want to skip this process.

Future pull request will be PR'ed to bb_develop branch.

Regards
Andre

PS: I would be happy to discuss this further on phone or something like that if you're interested. I'm a german native speaker if that's convenient for you :) And maybe I could clarify this a little better.

**Comment by matthias-schlimm** [comment]
Send you a pm on Twitter about further discussions

**Comment by EUCweb** [comment]
I'm change the Base Branch to BB_develop

**Comment by EUCweb** [comment]
I  will manually merge this PR into **BB_develop**

**Comment by EUCweb** [comment]
I'm changed the default branch to BB_develop, so all pull requests are made here now

File: ADMX/BaseImageScriptFramework.admx [hunk 1]
```diff
@@ -690,6 +690,7 @@
         </item>
       </disabledList>
       <elements>
+        <boolean id="CHK_P2V_SKIP_IMG" key="SOFTWARE\Policies\Login Consultants\BISF" valueName="LIC_BISF_CLI_P2V_SKIP_IMG" />
```
Comment (EUCweb) [inline ADMX/BaseImageScriptFramework.admx:693]: manually merged into BB_develop ADMX

**Comment by EUCweb** [comment]
manually merged into BB_develop, ref to https://github.com/EUCweb/BIS-F/issues/46

**Pull request closed without merging**